CMakeLists.txt

```diff
@@ -112,6 +118,20 @@ add_definitions(-DAMBER_ENGINE_VULKAN=$<BOOL:${Vulkan_FOUND}>)
 add_definitions(-DAMBER_ENGINE_DAWN=$<BOOL:${Dawn_FOUND}>)
 add_definitions(-DAMBER_ENABLE_SPIRV_TOOLS=$<BOOL:${AMBER_ENABLE_SPIRV_TOOLS}>)
 add_definitions(-DAMBER_ENABLE_SHADERC=$<BOOL:${AMBER_ENABLE_SHADERC}>)
+add_definitions(-DAMBER_ENABLE_DXC=$<BOOL:${AMBER_ENABLE_DXC}>)
+
+set(CMAKE_DEBUG_POSTFIX "")
+
+# This has to be done very early so the link path will get set correctly for all
+# the various libraries and binaries.
+if (${AMBER_ENABLE_DXC})
+  link_directories("${CMAKE_BINARY_DIR}/third_party/dxc/lib")
```

Comment:
For later:    We can use `<projectname>_BINARY_DIR` to specify this dir, without assuming its source was under third_party.

It's kind of weird, but DXC sets the project name to "LLVM", so then it would still look weird!
See https://github.com/microsoft/DirectXShaderCompiler/blob/master/CMakeLists.txt#L28